frontend/src/App.css

```
@@ -1487,6 +1579,16 @@ button:disabled {
   font-weight: 300;
 }
 
+.history-delete svg {
+  width: 16px;
+  height: 16px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)